messages/dji_msgs/msg/Links.msg

```diff
@@ -11,4 +11,12 @@ string HOME_POINT = 'home_point'
 string BASE_LINK = 'base_link'
 string BASE_FLAT = 'base_flat_link'
 string BASE_ENU = 'base_ENU_link'
+string GIMBAL_CAMERA_LINK = 'gimbal_camera_link'
+
+
+# Used by hook_estimator
+string CAMERA_LINK = 'camera_link'
+string WINCH_LINK = 'winch_link'
+string IMU_LINK = 'imu_link'
```

Comment:
why do you need to know _where_ the imu is? You should be using the fused estimate like here: https://github.com/smarc-project/smarc2/blob/3e0210ba74c582b49b2b2c03aef93c0e85346df9/behaviours/alars/alars/alars_recover_action_server.py#L383